### ISSUE TITLE
There is no functionality provided to adjust columns width which is achievable using single pointer: A11y_.NET CoreWinforms_ Data Binding Example for controls_DraggingMovement

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11433]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### Issue description

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG2.5.7; #A11yWCAG2.2#DesktopApp; #Win11; #FTP; 

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "DataBinding Example" and hit ENTER key.
4. "Data Binding Example for controls" screen would appear.
5. TAB to "List view" and "Grid" present.
6. Observe that There is no functionality provided to adjust columns width which is achievable using single pointer

### Actual Result: 
There is no functionality provided to adjust columns width which is achievable using single pointer.

**Similar issue is observed in throughout the application for column width.**

### Expected Result:
There should be functionality provided to adjust columns width which is achievable using single pointer.

### User Impact:
If the drag and drop functionality is not performing as expected on controls within a user interface, it can hamper user experience. Users may resort to alternative methods to achieve their goals; this increases the cognitive load and effort required to complete tasks. 

### Attachment

https://github.com/dotnet/winforms/assets/90901519/973fc41e-4274-41d2-8948-946bb6e9b370

**Comment by Olina-Zhang** [comment]
For DataGridView: we can use Alt + Left/Right arrow to narrows/expands width of dataGridView cell, but ListView cannot.

**Comment by merriemcgaw** [comment]
@PoojaNamde can your team please mark this for closure. I'll label all ready for closure as A11y-resolved.

**Comment by Yash14j** [comment]
Github Tags:#Rev:yaja;

**Comment by PoojaNamde** [comment]
@merriemcgaw Providing keyboard equivalence for a dragging operation does not meet this Success Criterion. there should be single pointer available. for ex -
[Fluent UI - Controls - React - DetailsList - Keyboard Column Reorder & Resize (microsoft.com)](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/keyboardcolumnreorderresize)

**Comment by merriemcgaw** [comment]
I see. I need to talk to SMEs to see what we can or should do here for this older control.

**Comment by merriemcgaw** [comment]
Office Hours determined that we should demonstrate on one DGV that this can be done via existing properties, but that it is not on the library to directly enable it. It is one the developer using the library to implement a solution.

@Olina-Zhang can your team implement a menu item that changes the size of a DGV column. You can do it on this DGV and for a couple columns only, this is just to show that it can be done.

**Comment by Olina-Zhang** [comment]
@merriemcgaw I added two menu items to change the size of DGV column and ListView column as following. Is that okay?

https://github.com/dotnet/winforms/assets/26474449/7b7194cd-1a7f-4871-9c19-50a69f1ff21b

**Comment by merriemcgaw** [comment]
I believe as long as they work properly, this should be sufficient for us from a UI library perspective. @PoojaNamde can verify that the functionality works and we should be able to close this. @PoojaNamde talk with me offline if you have questions.

**Comment by Olina-Zhang** [comment]
@PoojaNamde have updated and uploaded that app, please test it again. Thanks!

**Comment by PoojaNamde** [comment]
@Olina-Zhang I have downloaded the latest template, but I am not able to see the 2 menus newly added in template. Please find the below screenshot for reference.

![image](https://github.com/dotnet/winforms/assets/90901519/bb6ba5f8-33cc-4e83-8010-c779c93aa0c4)

**Comment by Olina-Zhang** [comment]
@PoojaNamde it should be as following, can you check again using the updated one?
![image](https://github.com/dotnet/winforms/assets/26474449/31b1910f-017e-4782-b15b-f9aba2c9dfa5)

**Comment by PoojaNamde** [comment]
@Olina-Zhang Reverified this issue in the latest env, here the fix is incomplete. With the buttons provided we are only able to adjust the width of 1st column and not for the others. Please find the more ways to fix this issue.

**A non-exhaustive list of techniques to pass this requirement is below.** 
 
1. In a table which can be rearranged by dragging columns, allow the order of data to be set in a separate “edit table” dialog. 
2. Allow a user to type values in an input field next to a volume slider, or include clickable arrows at either end. 
3. Include context menu options which can be accessed via right click. 

It should be noted that the requirement says that functionality must be available with a single pointer. It says nothing about needing to use the same component for both drag and non-drag methods of operation.

**Comment by PoojaNamde** [comment]
As per discussion with Ahmed, added two sample menu items to change the size of DGV column and List View column. Closing this bug. Please find below screenshot for reference.

![image](https://github.com/user-attachments/assets/1fb07e6f-f7cc-4cb9-b011-9e2daea2c973)


#Closed; #Regressed:08-09-24;
